mcs/class/System.Drawing/Test/System.Drawing.Drawing2D/PathGradientBrushTest.cs

```diff
@@ -36,7 +36,9 @@
 namespace MonoTests.System.Drawing.Drawing2D {
 
 	[TestFixture]
+#if !NET40
 	[SecurityPermission (SecurityAction.Deny, UnmanagedCode = true)]
+#endif
```

Comment:
OK, I just pushed a new commit which removes the CAS attributes in the unit tests.

Let me know if it's easier I prepare a seperate PR for that.